packages/appeals-service-api/src/lib/notify.js

```diff
@@ -0,0 +1,69 @@
+const { NotifyClient } = require('notifications-node-client');
+const fetch = require('node-fetch');
+const config = require('./config');
+const logger = require('./logger');
+const { getLpa } = require('../services/lpa.service');
+
+function getAddress(siteAddress) {
+  let address = '';
```

Comment:
Would it not be cleaned to do:

```
const address = []
if (...) {
  address.push()
}
return address.join('\n')
```

Would save having to modify the pointer reference

Reply:
OK